sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/HashAggregateExec.scala

```diff
@@ -466,10 +466,13 @@ case class HashAggregateExec(
       val resultVars = bindReferences[Expression](
         resultExpressions,
         inputAttrs).map(_.genCode(ctx))
+      val evaluateNondeterministicAggResults =
```

Comment:
There is `evaluateAggResults`. This `evaluateNondeterministicAggResults` looks confusing because they are not aggregation result like `evaluateAggResults`. 

Maybe `evaluateNondeterministicResults`.